beacon-chain/attestation/service.go

```diff
@@ -105,7 +105,7 @@ func (a *Service) LatestAttestation(ctx context.Context, index uint64) (*pb.Atte
 
 	// return error if validator has no attestation.
 	if _, exists := a.Store[pubKey]; !exists {
-		return nil, fmt.Errorf("validator index %d does not have an attestation", index)
+		return nil, nil
```

Comment:
Can you fix this comment here? Since it's technically not returning an error

Reply:
Can you return a grpc not found error?

Reply:
nvm this isnt grpc